docs/api/ref/api.yml

```diff
@@ -31,69 +31,49 @@ paths:
       tags:
         - Read Requests
       summary: Get information for a specific product by barcode
-      parameters:
-        - name: barcode
-          in: path
-          description: |
-            The barcode of the product to be fetched
-          required: true
-          style: simple
-          explode: false
-          schema:
-            type: string
-            example: "3017620422003"
-      responses:
-        "200":
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: ./responses/get_product_by_barcode.yaml
-              examples:
-                spread-example:
-                  $ref: ./examples/get_product_by_barcode_spread.yaml
-
       description: |
-        A product can be fetched via its unique barcode.
-        It returns all the details of that product response.
+        Fetches product details by its unique barcode. 
+        Can return all product details or specific fields like knowledge panels.
       operationId: get-product-by-barcode
-  "/api/v2/product/{barcode}?fields=knowledge_panels":
```

Comment:
Yep, when I did this I knew I was bending the rules of OpenAPI. In between we now have a [link to knowledge panels structure](https://openfoodfacts.github.io/openfoodfacts-server/api/ref-v2/#cmp--schemas-product-knowledge-panels), so it's really time to ditch it :slightly_smiling_face: